README.md

```diff
@@ -319,19 +335,33 @@ There will be an incentive of **13x** for those who claim first, reducing to **7
 
 <img src="img/gift_execution.png" alt="cyberGift Execution">
 
-## Final Distribution   
+## Final Distribution
 
-[Final Distribution Table](https://console.cloud.google.com/bigquery?project=cosmic-keep-223223&p=cosmic-keep-223223&d=final&t=final_distribution&page=table&ws=!1m5!1m4!4m3!1scosmic-keep-223223!2sfinal!3sfinal_distribution)  
+[Final Distribution Table](https://console.cloud.google.com/bigquery?project=cosmic-keep-223223&p=cosmic-keep-223223&d=final&t=final_distribution&page=table&ws=!1m5!1m4!4m3!1scosmic-keep-223223!2sfinal!3sfinal_distribution)
 
 **Check your address in:**
-- **web2** by the [Dashboard](https://datastudio.google.com/u/0/reporting/53e1c28b-9f10-497c-9b5b-9f2a4749450b)  
+
+- **web2** by the [Dashboard](https://datastudio.google.com/u/0/reporting/53e1c28b-9f10-497c-9b5b-9f2a4749450b)
 - **web3** by IPFS link
-```gateway.ipfs.cybernode.ai/ipfs/QmWvynJ9yrfU5ju8dRSDBY7SmfEugoTyhtA5YmvSqE6Q1c/<your address>.json```  
-for example [gateway.ipfs.cybernode.ai/ipfs/QmWvynJ9yrfU5ju8dRSDBY7SmfEugoTyhtA5YmvSqE6Q1c/0x91170c80ce048e00202e7ad72f8cfab86ec9004d.json](https://gateway.ipfs.cybernode.ai/ipfs/QmWvynJ9yrfU5ju8dRSDBY7SmfEugoTyhtA5YmvSqE6Q1c/0x91170c80ce048e00202e7ad72f8cfab86ec9004d.json)
-# Result Validation
-My name is Maxim Razhev (@ninjascant). I'm a software engineer with 3+ years of experience in blockchain analytics development. I made a manual re-check of queries used here e.g.
+  ```gateway.ipfs.cybernode.ai/ipfs/QmWvynJ9yrfU5ju8dRSDBY7SmfEugoTyhtA5YmvSqE6Q1c/<your address>.json```  
+  for
+  example [gateway.ipfs.cybernode.ai/ipfs/QmWvynJ9yrfU5ju8dRSDBY7SmfEugoTyhtA5YmvSqE6Q1c/0x91170c80ce048e00202e7ad72f8cfab86ec9004d.json](https://gateway.ipfs.cybernode.ai/ipfs/QmWvynJ9yrfU5ju8dRSDBY7SmfEugoTyhtA5YmvSqE6Q1c/0x91170c80ce048e00202e7ad72f8cfab86ec9004d.json)
+
+## Result Validation
+
+My name is Maxim Razhev (@ninjascant). I'm a software engineer with 3+ years of experience in blockchain analytics
+development. I made a manual re-check of queries used here e.g.
+
 - re-calculating balances on a subset of wallets and comparing with a node output
 - comparing the list of NFTs and other entities with open sources like Etherscan and OpenSea
 - verifying that the data presented in the final tables are consistent with query logic
 
-By making this commit, I confirm that the results of the analysis presented in this repository are checked by me and correct.
+By making this commit, I confirm that the results of the analysis presented in this repository are checked by me and
+correct.
+
+# Small Gift for Bostrom Birthday
+
+Since the main Gift cannot be unlocked until we reach the goal, the cyber~congress decided to make a small gift.  
+Soon each citizen who claim the gift will receive 1A and 1V by each claimed address.  
```

Comment:
Soon each citizen who claims the gift will receive 1A and 1V for each claimed address.